## Homework

1. Suponga dos eventos, A y B, y que P(A) = 0.50, P(B) = 0.60 y P(A ∩ B) = 0.40.<br>
- a. Halle P(A | B).
- b. Halle P(B | A).
- c. ¿A y B son independientes? ¿Por qué sí o por qué no?

In [15]:
pA, pB, pAB = 0.5, 0.6, 0.4
print(f'a) P(A|B)={(pAB/pB).__round__(4)}')
print(f'b) P(B|A)={pAB/pA}')
print(f'c) A y B son sucesos independientes: {pAB/pB == pA}')
'Para que fueran independientes, P(A|B) = P(A)'

a) P(A|B)=0.6667
b) P(B|A)=0.8
c) A y B son sucesos independientes: False


'Para que fueran independientes, P(A|B) = P(A)'

2. Suponga dos eventos, A y B, que son mutuamente excluyentes. Admita, además, que P(A) = 0.30 y P(B) = 0.40.<br>
- a. Obtenga P(A ∩ B). ¿Existe intersección entre los dos conjuntos?.
- b. Calcule P(A | B).
- c. Un estudiante de estadística argumenta que los conceptos de eventos mutuamente excluyentes y eventos independientes son en realidad lo mismo y que si los eventos son mutuamente excluyentes deben ser también independientes. ¿Está usted de acuerdo? Use la información sobre las probabilidades para justificar su respuesta.

In [3]:
pA, pB, pAB = 0.3, 0.4, 0
print(f'a) P(A ∩ B) = {pAB}. Hay intersección: {bool(pAB)}')
print(f'b) P(A | B) = {pAB/pA}. Eventos independientes: {bool(pAB/pA)}')
print(f'c) A y B son mutuamente excluyentes: {pAB == 0}. Son independientes: {bool(pAB/pA)}.')


a) P(A ∩ B) = 0. Hay intersección: False
b) P(A | B) = 0.0. Eventos independientes: False
c) A y B son mutuamente excluyentes: True. Son independientes: False.


3. Dada la siguiente tabla:<br>
![Sucesos](../_src/assets/sucesos.PNG)<br> 
Si en la concesionaria se seleccionan dos ventas con reposición (los sucesos son independientes), hallar la probabilidad de que las ventas sean:<br>
 
  - a. La primera de un comprador de “menos de 40 años” y la segunda de uno de "entre 40 y 50 años". 
  
  - b. Las dos sean de autos "nacionales".

In [17]:
primera = 30/80; segunda = 34/80; nacional = 50/80
print(f'a) La probabilidad es de: {(primera*segunda).__round__(4)}')
print(f'b) La probabilidad es de: {(nacional**2).__round__(4)}')


a) La probabilidad es de: 0.1594
b) La probabilidad es de: 0.3906


4. Si la selección de las dos ventas se realiza SIN reposición (los sucesos son condicionales), hallar la probabilidad de que las ventas sean:<br>

 - a. La primera de un comprador de “menos de 40 años” y la segunda de uno de "entre 40 y 50 años".

 - b. Las dos sean de autos "nacionales".

In [18]:
primera = 30/80; segunda = 34/79; nacional = 50/80; nacional2 = 49/79
print(f'a) La probabilidad es de: {(primera*segunda).__round__(4)}')
print(f'b) La probabilidad es de: {(nacional*nacional2).__round__(4)}')

a) La probabilidad es de: 0.1614
b) La probabilidad es de: 0.3877


5. Si la selección de las dos ventas se realiza SIN reposición (los sucesos son condicionales),  hallar la probabilidad de que las ventas sean:<br>
    
 - a. De un comprador de “menos de 40 años” y de uno de "entre 40 y 50 años", sin importar el orden.

In [19]:
casoA = (30/80) * (34/79); casoB = (34/80) * (30/79)
print(f'La probabilidad es de: {(casoA + casoB).__round__(4)}')

La probabilidad es de: 0.3228


 6. Debido al aumento de los costos de los seguros, en Estados Unidos 43 millones de personas no cuentan con un seguro médico. En la tabla siguiente se muestran datos muestrales representativos de la cantidad de personas que cuentan con seguro médico:<br>
 ![Seguro](../_src/assets/seguro.PNG)

- a. Con estos datos elabore una tabla de probabilidad conjunta y úsela para responder las preguntas restantes.
- b. ¿Qué indican las probabilidades marginales acerca de la edad de la población?
- c. ¿Cuál es la probabilidad de que una persona tomada en forma aleatoria no tenga seguro médico?
- d. Si la persona tiene entre 18 y 34 años, ¿cuál es la probabilidad de que no tenga seguro médico?
- e. Si la persona tiene 35 años o más, ¿cuál es la probabilidad de que no tenga seguro médico?
- f. Si la persona no tiene seguro médico, ¿cuál es la probabilidad de que tenga entre 18 y 34 años?

In [33]:
import pandas as pd
df = pd.DataFrame([[750,170],[950,130],[1700,300]],index=['18 a 34','35 o mayor','Total'],columns=['Si','No'])
df['Total'] = df.Si + df.No
df = df/2000
df

Si     No  Total
18 a 34     0.375  0.085   0.46
35 o mayor  0.475  0.065   0.54
Total       0.850  0.150   1.00

In [34]:
print('b) Indican la probabilidad de que cada elemento pertenezca a uno de los rangos etarios.')
print(f'c) La probabilidad es: {df.No[2]}')
print(f'd) La probabilidad es: {(df.No[0]/df.Total[0]).__round__(4)}')
print(f'e) La probabilidad es: {(df.No[1]/df.Total[1]).__round__(4)}')
print(f'f) La probabilidad es: {(df.No[0]/df.No[2]).__round__(4)}')

b) Indican la probabilidad de que cada elemento pertenezca a uno de los rangos etarios.
c) La probabilidad es: 0.15
d) La probabilidad es: 0.1848
e) La probabilidad es: 0.1204
f) La probabilidad es: 0.5667


#### Puedes complementar con los ejercicios del 33 al 40 del Capitula 4 del libro "ESTADÍSTICA PARA ADMINISTRACIÓN Y ECONOMÍA"

33. Una muestra de estudiantes de la maestría en administración de negocios, arrojó la siguiente información sobre la principal razón que tuvieron los estudiantes para elegir la escuela en donde hacen sus estudios.

(Grafico en pagina 169)

- a. Con estos datos elabore una tabla de probabilidad conjunta.
- b. Use las probabilidades marginales: calidad de la escuela, costo de la escuela y otras para comentar cuál es la principal razón por la que eligen una escuela.
- c. Si es un estudiante de tiempo completo, ¿cuál es la probabilidad de que la principal razón para su elección de la escuela haya sido la calidad de la escuela?
- d. Si es un estudiante de medio tiempo, ¿cuál es la probabilidad de que la principal razón para su elección de la escuela haya sido la calidad de la escuela?
- e. Si A denota el evento es estudiante de tiempo completo y B denota el evento la calidad de la escuela fue la primera razón para su elección, ¿son independientes los eventos A y B? Justifique su respuesta.

Razones de su elección
                            Calidad de  Costo de
                            la escuela  la escuela  Otras   Totales
Tipo de     Tiempo completo 421         393         76      890
estudiante  Medio tiempo    400         593         46      1039
            Totales         821         986         122     1929

In [50]:
df_e = pd.DataFrame([[421,393,76,890],[400,593,46,1039],[821,986,122,1929]],index=['FT','PT','Tot'],columns=['Calidad','Costo','Otro','Total'])
df_e = df_e/1929
df_e

Calidad     Costo      Otro     Total
FT   0.218248  0.203733  0.039399  0.461379
PT   0.207361  0.307413  0.023847  0.538621
Tot  0.425609  0.511146  0.063245  1.000000

In [37]:
print('b) Lo más probable es que un alumno elija por costo')
print('c)',(df_e.Calidad[0]/df_e.Total[0]).round(4))
print('d)',(df_e.Calidad[1]/df_e.Total[1]).round(4))
print(f'''e) Independencia: P(A ∩ B) = P(A)*P(B)
   P(A ∩ B) = {df_e.Calidad[0].round(4)}, P(A) = {df_e.Total[0].round(4)}, P(B) = {df_e.Calidad[2].round(4)}
   Son independientes: {df_e.Calidad[0] == df_e.Total[0]*df_e.Calidad[2]}''')

b) Lo más probable es que un alumno elija por costo
c) 0.473
d) 0.385
e) Independencia: P(A ∩ B) = P(A)*P(B)
   P(A ∩ B) = 0.2182, P(A) = 0.4614, P(B) = 0.4256
   Son independientes: False


34. La tabla siguiente muestra las probabilidades de los distintos tipos sanguíneo en la población.

(Gráfico en página 170)
- a. ¿Cuál es la probabilidad de que una persona tenga sangre tipo O?
- b. ¿De que tenga sangre Rh-?
- c. ¿Cuál es la probabilidad de que una persona sea Rh- dado que la persona tiene sangre tipo O?
- d. ¿Cuál es la probabilidad de que una persona tenga sangre tipo B dado que es Rh+?
- e. ¿Cuál es la probabilidad de que en un matrimonio, los dos sean Rh-?
- f. ¿Cuál es la probabilidad de que en un matrimonio, los dos tengan sangre AB?

    A       B       AB      O
Rh+ 0.34    0.09    0.04    0.38
Rh- 0.06    0.02    0.01    0.06

In [39]:
print('a)',0.38+0.06)
print('b)',0.06+0.02+0.01+0.06)
print('c)',(0.06/(0.06+0.38)).__round__(4))
print('d)',(0.09/(0.34+0.09+0.04+0.38)).__round__(4))
print('e)',(0.06+0.02+0.01+0.06)**2)
print('f)',((0.04+0.01)**2).__round__(4))


a) 0.44
b) 0.15
c) 0.1364
d) 0.1059
e) 0.0225
f) 0.0025


35. El Departamento de Estadística Laboral de Estados Unidos reúne datos sobre las ocupaciones de las personas entre 25 y 64 años. La tabla siguiente presenta el número de hombres y mujeres (en millones) en cada una de las categorías ocupacionales.

(Gráfico en página 170)

- a. Desarrolle una tabla de probabilidad conjunta.
- b. ¿Cuál es la probabilidad de que un trabajador mujer sea directivo o profesional?
- c. ¿Cuál es la probabilidad de que un trabajador hombre esté en producción con precisión?
- d. ¿Es la ocupación independiente del género? Justifique su respuesta con el cálculo de la probabilidad.

                                        H   M
Directivo                           19 079 19 021
Enseñanza/Ventas/ Administrativo    11 079 19 315
Servicio                             4 977  7 947
Producción con precisión            11 682  1 138
Operadores/Obrero                   10 576  3 482
Agricultura/Ganadería/               1 838    514

In [52]:
df_o = pd.DataFrame([[19079,19021],[11079,19315],[4977,7947],[11682,1138],[10576,3482],[1838,514], [0,0]],\
    index=['Directivo/Profesional','Enseñanza/Ventas/Administrativo','Servicio','Producción con precisión','Operador/Obrero','Agricultura/Ganadería/Pesca','Total'],\
                    columns=['Hombres','Mujeres'])
df_o.loc['Total'] = [df_o.Hombres.sum(),df_o.Mujeres.sum()]
df_o['Total']= df_o.Hombres + df_o.Mujeres
df_o = df_o/df_o.Total[6]
df_o

Hombres   Mujeres     Total
Directivo/Profesional            0.172430  0.171906  0.344335
Enseñanza/Ventas/Administrativo  0.100128  0.174563  0.274691
Servicio                         0.044980  0.071822  0.116803
Producción con precisión         0.105578  0.010285  0.115863
Operador/Obrero                  0.095582  0.031469  0.127052
Agricultura/Ganadería/Pesca      0.016611  0.004645  0.021257
Total                            0.535310  0.464690  1.000000

In [59]:
print('b)', (df_o.Mujeres[0]/df_o.Total[0]).round(4))
print('c)', (df_o.Hombres[3]/df_o.Total[3]).round(4))
print(f'''d) Independencia: P(A ∩ B) = P(A)*P(B)
   P(A ∩ B) = {df_o.Hombres[3].round(4)}, P(A) = {df_o.Total[3].round(4)}, P(B) = {df_o.Hombres[6].round(4)}
   Son independientes: {df_o.Hombres[3] == df_o.Total[3]*df_o.Hombres[6]}''')

b) 0.4992
c) 0.9112
d) Independencia: P(A ∩ B) = P(A)*P(B)
   P(A ∩ B) = 0.1056, P(A) = 0.1159, P(B) = 0.5353
   Son independientes: False


36. Reggie Miller de los Indiana Pacers tiene el record de la National Basketball Association de máscanastas de 3 puntos anotadas en toda una carrera, acertando en 89% de sus tiros (USA Today, 22 de enero de 2004). Suponga que ya casi al final de un juego cometen una falta contra él y le conceden dos tiros.

- a. ¿Cuál es la probabilidad de que acierte en los dos tiros?
- b. ¿De que acierte en por lo menos uno de los dos tiros?
- c. ¿De que no acierte en ninguno de los dos tiros?
- d. Al final de un juego de básquetbol suele ocurrir que cometan faltas contra un jugador del equipo opuesto para detener el reloj del juego. La estrategia usual es cometer una falta contra el peor tirador del otro equipo. Suponga que el centro de los Indiana Pacers acierta 58% de sus tiros. Calcule para él las probabilidades calculadas en los incisos a, b y c y muestre que hacer una falta intencional contra el centro de los Indiana Pacers es mejor que hacerlo contra Reggie Miller.

In [69]:
print('a)', (0.89**2).__round__(4))
print('b)', (1 - 0.11**2).__round__(4))
print('c)', (0.11**2).__round__(4))
print(f'd) Ambos tiros: {(0.58**2).__round__(4)}, al menos uno: {(1 - 0.42**2).__round__(4)}, fallar ambos: {(0.42**2).__round__(4)}')

a) 0.7921
b) 0.9879
c) 0.0121
d) Ambos tiros: 0.3364, al menos uno: 0.8236, fallar ambos: 0.1764


37. Visa Card de Estados Unidos estudia con qué frecuencia usan sus tarjetas (de débito y de crédito) los consumidores jóvenes, entre 18 y 24 años. Los resultados del estudio proporcionan las probabilidades siguientes.

• La probabilidad de que un consumidor use su tarjeta al hacer una compra es 0.37.

• Dado que un consumidor usa su tarjeta, la probabilidad de que tenga entre 18 y 24 años es 0.19.

• Puesto que un consumidor usa su tarjeta, la probabilidad de que sea mayor de 24 años es 0.81.

Datos de la Oficina de Censos de Estados Unidos indican que 14% de los consumidores tienen entre 18 y 24 años.

- a. Ya que un consumidor tiene entre 18 y 24 años, ¿cuál es la probabilidad de que use su tarjeta?
- b. Dado que un consumidor tiene más de 24 años, ¿cuál es la probabilidad de que use su tarjeta?
- c. ¿Qué interpretación se le da a las probabilidades de los incisos a y b?
- d. ¿Empresas como Visa, Master Card y Discover deben proporcionar tarjetas a los consumidores entre 18 y 24 años, antes de que tengan una historia crediticia? Si no, explique. Si sí, ¿qué restricciones deben poner las empresas a estos consumidores?

38. En un estudio de Morgan Stanley Consumer Research se muestrearon hombres y mujeres y se les preguntó qué preferían tomar: agua de botella o una bebida deportiva como Gatorade o Propel Fitness (The Atlanta Journal-Constitution, 28 de diciembre de 2005). Suponga que en el estudio hayan participado 200 hombres y 200 mujeres y que de todos 280 hayan preferido el agua de botella. En el grupo de los que preferían bebidas deportivas, 80 eran hombres y 40 eran mujeres. Sea:

M = el evento el consumidor es hombre

W = el evento el consumidor es mujer

B = el evento el consumidor prefiere agua de botella

S = el evento el consumidor prefiere una bebida deportiva

- a. ¿Cuál es la probabilidad de que en este estudio una persona prefiera agua de botella?
- b. ¿De que en este estudio una persona prefiera una bebida deportiva?
- c. ¿Cuáles son las probabilidades condicionales P(M | S) y P(W | S)?
- d. ¿Cuáles son las probabilidades conjuntas P(M ∩ S) y P(W ∩ S)?
- e. Dado que un consumidor es hombre, ¿cuál es la probabilidad de que prefiera una bebida deportiva?
- f. Ya que un consumidor es mujer, ¿cuál es la probabilidad de que prefiera una bebida deportiva?
- g. ¿Depende la preferencia por una bebida deportiva de que el consumidor sea hombre o mujer? Explique usando la información sobre las probabilidades.

39. Las probabilidades previas de los eventos A1 y A2 son P(A1) = 0.40 y P(A2) = 0.60. Sabe también que P(A1 ∩ A2) = 0. Suponga que P(B | A1) =  0.20 y P(B | A2) = 0.05.
- a. ¿A1 y A2 son eventos mutuamente excluyentes? Explique.
- b. Calcule P(A1 ∩ B) y P(A2 ∩ B).
- c. Calcule P(B).
- d. Emplee el teorema de Bayes para calcular P(A1 | B) y P(A2 | B).

40. Las probabilidades previas de los eventos A1, A2 y A3 son P(A1) = 0.20, P(A2) = 0.50 y P(A3) = 0.30. Las probabilidades condicionales del evento B dados los eventos A1,A2 y A3 son P(B | A1) = 0.50, P(B | A2) = 0.40 y P(B | A3) = 0.30.
- a. Calcule P(B ∩ A1), P(B ∩ A2) y P(B ∩ A3).
- b. Emplee el teorema de Bayes, ecuación (4.19), para calcular la probabilidad posterior P(A2 | B).
- c. Use el método tabular para emplear el teorema de Bayes en el cálculo de P(A1 | B), P(A2 | B) y P(A3 | B).